code/modules/mob/living/carbon/alien/death.dm

```diff
@@ -14,6 +14,14 @@
 
 	playsound(src.loc, 'sound/goonstation/effects/gib.ogg', 50, 1)
 
+	if(LAZYLEN(stomach_contents)) //Release eaten mobs when Beno is gibbed
+		for(var/mob/M in src)
+			LAZYREMOVE(stomach_contents, M)
```

Comment:
shouldnt this loop through `stomach_contents` in most cases this will be the same, but I can imagine some weird edge case where this could break it.